cqrs/src/memory.rs

```diff
@@ -84,33 +84,27 @@ where
         let result = stream.map(|stream| {
             let stream = stream.read();
             match (since, max_count) {
-                (Since::BeginningOfStream, None) => stream
-                    .events
-                    .iter()
-                    .map(ToOwned::to_owned)
-                    .map(Ok)
-                    .collect(),
+                (Since::BeginningOfStream, None) => {
+                    stream.events.iter().map(ToOwned::to_owned).collect()
```

Comment:
For consistency wanna split this back out to several lines?

Reply:
rustfmt disagrees.